src/futil.js

```diff
@@ -3,4 +3,34 @@ import F from 'futil'
 
 export let flattenProp = _.curry((prop, target) =>
   _.flow(F.expandObject(_.get(prop)), _.unset(prop))(target)
-)
\ No newline at end of file
+)
+
```

Comment:
Ideally all of these functions should have tests.  I know this has been waiting for a long time now so I'm tempted to let it slide, but I could see us just moving all of this to futil eventually, where we would DEFINITELY want tests for them.  I'll let @daedalus28 make that call since you both co-opted this 🙂